app/services/google_services.rb

```diff
@@ -2,11 +2,19 @@ class GoogleService
   attr_reader :google_rating
 
   def initialize(phone_number)
-    return nil if phone_number == nil
+    @google_rating = 1
+    if phone_number == nil
+      return @google_rating
+    end
     number = phone_number[1..-1]
     api = Faraday.new("https://maps.googleapis.com/maps/api/place/findplacefromtext/json?input=%2B#{number}&inputtype=phonenumber&fields=rating&key=#{ENV["GOOGLE_API_KEY"]}")
     response = api.get
     google = JSON.parse(response.body, symbolize_names: true)
-    @google_rating = google[:candidates][0][:rating].to_s
+    if google[:status] == "INVALID_REQUEST"
+      rating = 1
+    else
+      rating = google[:candidates][0][:rating].to_s
+    end
+    @google_rating = rating
```

Comment:
If we refactored we could get rid of this instance variable but its working great now